client/src/components/IngredientTags.js

```diff
@@ -5,7 +5,7 @@ import { FaMinus } from 'react-icons/fa'
 export default function IngredientTags({ ingredients, onDeleteTag, onToggleStatus }) {
     return (
         <Wrapper>
-            {ingredients.map((ingredient) =>
+            {ingredients && ingredients.map((ingredient) =>
```

Comment:
I think you don't need to check if ingredients is true since it's always an array and therefore always true :-).

Reply:
You were right, it worked just fine :). Thank you!!